get_active_hooks.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+"""
+    Report on Service & Web hooks for organization.
+"""
+import argparse
+import client
+import logging
+import urlparse
+import yaml
+
+"""
+    Lore:   swapping hook.test for hook.ping will cause repetition of the
+            actions.  In particular, a number of repos post to IRC channels
+            and/or bugs on commits, so expect comments to that effect.
+"""
+
+logger = logging.getLogger(__name__)
+
+
+def report_hooks(gh, org, active_only, unique_only, do_ping, yaml_out):
+    org_handle = gh.organization(org)
+    org_struct = org_handle.to_json()
+    repo_list = []
+    org_struct['repo_list'] = repo_list
+    unique_hooks = set()
+    msg = "Active" if active_only else "All"
+    for repo in org_handle.iter_repos():
+        repo_struct = repo.to_json()
+        hook_list = []
+        repo_struct['hook_list'] = hook_list
+        repo_list.append(repo_struct)
```

Comment:
I feel like you can do these three/four steps in one explicitly and it won't be as hard to understand